### PR TITLE
[INLONG-7912][Manager] Only issue normal DataProxy nodes

### LINKED ISSUE #7912: [Improve][Manager] Only response DataProxy nodes in normal status
### Description

manager only issues normal dataproxy node information.
企业微信截图_b1f5e800-ca2e-4c3c-b48c-dc71891c3bdd
企业微信截图_5465dcef-af1a-4c80-80bb-a88373afbd6f

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #7912 

### Motivation

manager only issues normal dataproxy node information.

### Modifications

manager only issues normal dataproxy node information.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java [hunk 1]
```diff
@@ -1154,6 +1155,11 @@ public DataProxyNodeResponse getDataProxyNodes(String groupId, String protocolTy
         // TODO consider the data proxy load and re-balance
         List<DataProxyNodeInfo> nodeList = new ArrayList<>();
         for (InlongClusterNodeEntity nodeEntity : nodeEntities) {
+            if (Objects.equals(nodeEntity.getStatus(), NodeStatus.HEARTBEAT_TIMEOUT.getStatus())) {
+                LOGGER.debug("data proxy is heart time out by parentId={}, ip={}, port={}", nodeEntity.getParentId(),
+                        nodeEntity.getIp(), nodeEntity.getPort());
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java:1160]:
```suggestion
                LOGGER.debug("dataproxy node was timeout, parentId={} ip={} port={}",
                        nodeEntity.getParentId(), nodeEntity.getIp(), nodeEntity.getPort());
```

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java]: done

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Comment by woofyzhao** [comment]
+1

**Pull request merged**